src/output_lcd/lcd_driver.c

```diff
@@ -381,7 +381,17 @@ static esp_err_t s3_lcd_configure_gpio()
     return ESP_OK;
 }
 
-void epd_lcd_init(const LcdEpdConfig_t* config, int display_width, int display_height) {
+void IRAM_ATTR epd_lcd_init(const LcdEpdConfig_t* config, int display_width, int display_height) {
+
+    if (CONFIG_ESP32S3_DATA_CACHE_LINE_SIZE < 64) {
```

Comment:
This looks very useful to me. Instead of the cache override I would mention that displays.c bus_speed should be lower to work with a line size <64 bytes